src/models/Popup.js

```diff
@@ -452,6 +523,10 @@ export default class Popup extends Base {
     // returns the parent element with the specified class, or the original element - null if not found
     const btn = traverseDOMPath(event.target, 'cc-btn') || event.target
     
+    if (btn.classList.contains( 'cc-btn' ) && btn.classList.contains( 'cc-customize' )){
+      this.showCustomizeSettingsContent();
+      return
+    }
```

Comment:
Please space unrelated ifs so the conditions are not wrongfully grouped.

```suggestion
    }

```